functors_task_1/student/include/impl.h

```diff
@@ -8,6 +8,15 @@
  */
 struct BeerOrganizer
 {
+private:
+    int next_beer = static_cast<int>(BeerBrand::HoeGaarden);
```

Comment:
_why don't use none and max?_

Reply:
I thought we are supposed to start with "HoeGarden" as a valid name of brand. Is not ok my implementation?

Reply:
Yeah, so you should shift the range accordingly, as we use other boundaries